### DISCUSSION TITLE
[@types/lodash] cannot find source code for multiple versions

### DISCUSSION THREAD

**Discussion by varunmankal** [DefinitelyTyped/DefinitelyTyped#57203] [Issues with a @types package]
where can I find the source code for a specific version ? For example: @types/lodash@4.14.176, @types/body-parser@1.19.1, @types/connect@3.4.35

[a comment by an automated account is not shown]

**Comment by RyanCavanaugh** [comment]
The source code is in the npm package, so I'm not really sure what the question is?

**Reply by varunmankal** [reply]
I mean where can I find the source code in github for a specific version. For example, github source code for the package browserslist@4.17.4 can be found here: https://github.com/browserslist/browserslist/tree/4.17.4

**Reply by RyanCavanaugh** [reply]
There's no automated tracking at that granularity
